xla/hlo/tools/tests/hlo_opt_dump.hlo

```diff
@@ -1,7 +1,21 @@
+// === Checking dump with dump for passes disabled ===
```

Comment:
I think it's worth testing the `--debug_options_file` option in a separate test file. @abhigunj, let me know if you disagree

Reply:
Done.